src/marqo/tensor_search/tensor_search.py

```diff
@@ -1519,20 +1632,21 @@ def construct_vector_input_batches(query: Optional[Union[str, Dict]], index_info
         A tuple of string batches. The first is text content the second is image content.
     """
     # TODO - infer this from model
-    treat_urls_as_images = True
+    treat_urls_as_media = True
+
     if isinstance(query, str):
-        if treat_urls_as_images and _is_image(query):
+        if treat_urls_as_media and _is_image(query):
```

Comment:
Are we not supporting audio/video as search query?

Reply:
we do, that's why it is treat_urls_as_media. I made a test for video search.

Reply:
ah but the _is_image check is redundant right? 

It probably works because _is_image only really checks if it is a valid url so this passes. I will change it to something clearer

Reply:
done